conf/php/php-fpm-73.conf

```diff
@@ -45,7 +45,7 @@ error_log = var/log/error.log
 ; Log level
 ; Possible Values: alert, error, warning, notice, debug
 ; Default Value: notice
-;log_level = notice
+log_level = warning
```

Comment:
Would it be possible to make that configurable ?

Reply:
It is already through the normal process (including files or statement in composer.json)